tfopgen/templates/Makefile.j2

```diff
@@ -21,12 +22,12 @@ OBJECTS=$(addsuffix .o, $(basename $(SOURCES)))
 LIBRARY={{shared_library}}
 
 # Compiler flags
-INCLUDES= -I $(TF_INC)
+INCLUDES= -I $(TF_INC) -I$(TF_INC)/external/nsync/public
 CPPFLAGS=-std=c++11 $(TF_FLAGS) $(INCLUDES) -fPIC -fopenmp -O2 -march=native -mtune=native
 NVCCFLAGS=-std=c++11 -D GOOGLE_CUDA=$(TF_CUDA) $(TF_FLAGS) $(INCLUDES) \
 	-x cu --compiler-options "-fPIC" --gpu-architecture=sm_30 -lineinfo
 
-LDFLAGS = -fPIC -fopenmp
+LDFLAGS = -fPIC -fopenmp -L$(TF_LIB) -ltensorflow_framework{{extra_libs}}
 
```

Comment:
One possible way of handling the the presence or absence of the `-ltensorflow_framework`   library would be to add `tensorflow_version` to the YAML.